### PR TITLE
Docs: Update dashboard URL variables docs

### LINKED ISSUE #80249: Dashboard URL variables: clarify intent of
The intent of this page needs to be made clearer along with other minor fixes.

### Tasks
- [x] Fix missing link in intro
- [x] Change headings to non-gerund version of the verb
- [x] Clarify if this is telling you how to add variables to a dashboard URL or if you create dashboard URLs while there variables are in context, then this is how they'll be represented (maybe both?)

### PR DESCRIPTION
**Pull request by imatwawana**
Clarifies intent of the content
Creates shared file for time range control content that's on two pages and replaces that text in both files
Overall style edit

### CODE REVIEW

File: docs/sources/shared/dashboards/time-range-URLs.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: Time range URLs
+comments: |
+  This file is used in the following files: dashboards/build-dashboards/create-dashboard-url-variables/index.md, dashboards/use-dashboards/index.md
+---
+
+You can control the time range of a dashboard by providing the following query parameters in the dashboard URL:
+
+- `from` - Defines the lower limit of the time range, specified in ms, epoch, or relative time.
+- `to` - Defines the upper limit of the time range, specified in ms, epoch, or relative time.
+- `time` and `time.window` - Defines a time range from `time-time.window/2` to `time+time.window/2`. Both parameters should be specified in `ms`. For example `?time=1500000000000&time.window=10000` results in a 10-second time range from 1499999995000 to 1500000005000`.
```
Comment (ivanortegaalba) [inline docs/sources/shared/dashboards/time-range-URLs.md:11]:
```suggestion
- `time` and `time.window` - Defines a time range from `time-time.window/2` to `time+time.window/2`. Both parameters should be specified in `ms`. For example `?time=1500000000000&time.window=10000` results in a 10-second time range from 1499999995000 to 1500000005000`.
- `timezone` - Defines the time zone. For example `timezone=Europe/Madrid`.
```

We have also the `timezone` parameter to override the default timezone of the dashboard

**Review by ivanortegaalba — APPROVED** [review]
@imatwawana I added the parameter for the timezone. The rest looks amazing! 💯

File: docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md [hunk 2]
```diff
@@ -68,34 +78,40 @@ Grafana interprets `var-example=value1&var-example=value2` as the dashboard vari
...
+[This dashboard in Grafana Play](https://play.grafana.org/d/000000074/alerting?var-app=backend&var-server=backend_01&var-server=backend_03&var-interval=1h) passes the variable `server` with multiple values, and the variables `app` and `interval` with a single value each.
 
-Grafana can add variables to dashboard links when you generate them from a dashboard's settings. For more information and steps to add variables, refer to [Manage dashboard links](ref:manage-dashboard-links).
+## Ad hoc filters
 
-## Passing ad hoc filters
+Ad hoc filters apply key/value filters to all metric queries that use a specified data source. For more information, refer to [Add ad hoc filters](ref:ad-hoc-filters).
 
-Ad hoc filters apply key/value filters to all metric queries that use a specified data source. For more information, refer to [Add ad hoc filters](ref:add-ad-hoc-filters).
+<!-- I'm not sure what the "a specified data source" part means; is this the dashboard data source or the ad hoc filter data source? -->
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md:87]:
```suggestion
```

File: docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md [hunk 3]
```diff
@@ -68,34 +78,40 @@ Grafana interprets `var-example=value1&var-example=value2` as the dashboard vari
...
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md:88]:
```suggestion
```

File: docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md [hunk 4]
```diff
@@ -68,34 +78,40 @@ Grafana interprets `var-example=value1&var-example=value2` as the dashboard vari
...
-This example in [Grafana Play](https://play.grafana.org/d/000000074/alerting?var-app=backend&var-server=backend_01&var-server=backend_03&var-interval=1h) passes the variable `server` with multiple values, and the variables `app` and `interval` with a single value each.
-
-## Adding variables to dashboard links
+[This dashboard in Grafana Play](https://play.grafana.org/d/000000074/alerting?var-app=backend&var-server=backend_01&var-server=backend_03&var-interval=1h) passes the variable `server` with multiple values, and the variables `app` and `interval` with a single value each.
 
-Grafana can add variables to dashboard links when you generate them from a dashboard's settings. For more information and steps to add variables, refer to [Manage dashboard links](ref:manage-dashboard-links).
+## Ad hoc filters
 
-## Passing ad hoc filters
+Ad hoc filters apply key/value filters to all metric queries that use a specified data source. For more information, refer to [Add ad hoc filters](ref:ad-hoc-filters).
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md:85]:
```suggestion
Ad hoc filters apply key/value filters to all metric queries that use the specified data source. For more information, refer to [Add ad hoc filters](ref:ad-hoc-filters).
```

**Pull request merged**